src/test/regress/expected/create_null_dist_key.out

```diff
@@ -1543,16 +1567,15 @@ CREATE TABLE trigger_table_1 (value int);
 CREATE TRIGGER trigger_1
 BEFORE INSERT ON trigger_table_1
 FOR EACH ROW EXECUTE FUNCTION increment_value();
-SELECT create_distributed_table('trigger_table_1', NULL, distribution_type=>null);
-ERROR:  cannot distribute relation "trigger_table_1" because it has triggers
-HINT:  Consider dropping all the triggers on "trigger_table_1" and retry.
-SET citus.enable_unsafe_triggers TO ON;
 SELECT create_distributed_table('trigger_table_1', NULL, distribution_type=>null);
  create_distributed_table
 ---------------------------------------------------------------------
 
 (1 row)
 
```

Comment:
Why are we removing this? It was useful to see that create_distributed_table() fails for the tables that have triggers.